docs/user/dashboard/dashboard.asciidoc

```diff
@@ -3,520 +3,286 @@
 
-* Enter the data field name in *Search field names*.
-* Click *Field filters*, then select the filter.
+* To start with an empty dashboard, click *Create dashboard*.
 
-[float]
-[[view-data-summaries]]
-==== View data field summaries
-
-To help you decide exactly the data you want to display, get a quick summary of each data field. The summary shows the distribution of
-values within the specified time range.
-
-To view the data field summary information, navigate to the field, then click *i*.
-
-[role="screenshot"]
-image::images/lens_data_info.png[Data summary window]
-
-*Lens* shows a summary depending on the type of data: date fields show the time distribution, string fields show the top 10 values, and numeric fields show a detailed summary with the top 10 values and a value distribution.
-
-[role="screenshot"]
-image::images/lens_data_info_documents.png[Data summary analyzed documents]
-
-*Lens* uses a sample of 5,000 documents to perform the field analysis. The bottom line of the summary shows the percentage of sampled documents over all available documents.
-
-When *Lens* presents the top 10 values distribution, it also shows the percentage of "Other" values. For array value fields, the percentage distribution is considers each value in the array separate.
-
-[role="screenshot"]
-image::images/lens_data_info_other.png[Data summary window with Other]
-
-NOTE: the sum of all the entries and "Other" may be above 100% by a small amount, that is due to a rounding problem when presenting all values as integers.
+* To open an existing dashboard, click the dashboard *Title* you want to open.
 
 [float]
-[[change-the-visualization-type]]
-==== Change the visualization type
-
-Use the automatically generated suggestions to change the visualization type, or manually select the type of visualization you want to view.
+=== Create panels
 
-*Suggestions* are shortcuts to alternative visualizations that *Lens* generates for you.
+Choose the type of panel you want to create, then save the panel to the dashboard.
 
-[role="screenshot"]
-image::images/lens_suggestions.gif[Visualization suggestions]
-
-If you’d like to use a visualization type outside of the suggestions, click the visualization type, then select a new one.
+. To create a panel, click *Create panel*, then click the panel type on the *New visualization* window.
++
+To add a saved panel, click *Add from library*, then select the panel you want to add. When a panel contains a stored query, both queries are applied.
 
-[role="screenshot"]
-image::images/lens_viz_types.png[]
+. To save the panel, click *Save* in the toolbar, then configure the *Save visualization* options.
 
-When there is an exclamation point (!) next to a visualization type, *Lens* is unable to transfer your data, but still allows you to make the change.
+.. Enter the *Title* and optional *Description*.
 
-[float]
-[[customize-the-data]]
-==== Customize the data
+.. From the *Tags* drop down, select any applicable tags. 
```

Comment:
or create a new tag.  Maybe link to tags doc?

Reply:
This section is not about creating tags. I would like to link to the tags doc, but that doc is currently missing step-by-step instructions on how to create a tag.